fbcore/src/main/java/com/facebook/common/activitylistener/ActivityListenerManager.java

```diff
@@ -32,13 +32,12 @@
   public static void register(
       ActivityListener activityListener,
       Context context) {
-    if (!(context instanceof ListenableActivity) && context instanceof ContextWrapper) {
-      context = ((ContextWrapper) context).getBaseContext();
-    }
     if (context instanceof ListenableActivity) {
       ListenableActivity listenableActivity = (ListenableActivity) context;
       Listener listener = new Listener(activityListener);
       listenableActivity.addActivityListener(listener);
+    } else if (context instanceof ContextWrapper) {
```

Comment:
This is not the same as the original behavior. If a ListenableActivity is wrapped in a ContextWrapper, the listener won't get added in this case.